packages/tinacms/src/admin/pages/CollectionListPage.tsx

```diff
@@ -284,10 +300,17 @@ const CollectionListPage = () => {
                 const documents = collection.documents.edges
                 const admin: TinaAdminApi = cms.api.admin
                 const pageInfo = collection.documents.pageInfo
-                const fields = collectionExtra.fields?.filter((x) =>
+
+                // get unique fields from all templates
+                const fields = (
```

Comment:
I'm surprised these fields don't need to be namespaced by their template name. 🤷 if it works, 👍

Reply:
I wonder if they may need to if you supplied a strict "starts_with" in the search? Otherwise filtering/search seemed to work fine for me but I'll do some extra testing on the beta release.